src/azure-cli/azure/cli/command_modules/iot/tests/latest/test_iot_commands.py

```diff
@@ -399,13 +399,13 @@ def test_identity_hub(self, resource_group, resource_group_location, storage_acc
         location = resource_group_location
 
         private_endpoint_type = 'Microsoft.Devices/IoTHubs'
-        identity_hub = 'identity-test-hub-cli'
+        identity_hub = 'identity-test-hub-cli20200720'
         identity_based_auth = 'identityBased'
-        event_hub_identity_endpoint_name = 'EventHubIdentityEndpoint'
+        event_hub_identity_endpoint_name = 'EventHubIdentityEndpoint20200720'
 
-        containerName = 'iothubcontainer1'
+        containerName = 'iothubcontainer20200720'
         storageConnectionString = self._get_azurestorage_connectionstring(rg, containerName, storage_account)
-        endpoint_name = 'Event1'
+        endpoint_name = 'Event20200720'
```

Comment:
Why change these names

Reply:
The test was failing because of the unavailability of the resources, so added the current date as a suffix to get a pass

Reply:
why is identity-test-hub-cli20200720 available while identity-test-hub-cli isn't available?
Is identity_hub pre-created before this test or created in this test?

---
In reply to: [458515573](https://github.com/Azure/azure-cli/pull/14449#discussion_r458515573) [](ancestors = 458515573)

Reply:
> The test was failing because of the unavailability of the resources, so added the current date as a suffix to get a pass

May I ask if the test failure due to the conflict of repeatedly creating resources with the same name? If so, could you change the name to random?